src/analyses/basePriv.ml

```diff
@@ -103,22 +103,23 @@ struct
         {st with cpa = CPA.add x v st.cpa}
     )
 
-  let sync ask getg (st: BaseComponents (D).t) reason =
+  let sync ask getg sideg (st: BaseComponents (D).t) reason =
     (* For each global variable, we create the diff *)
```

Comment:
This comment seems outdated? Maybe also it should not be `add_var` anymore but `side_effect_var` or sth?